service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/BaseDevice.java

```diff
@@ -340,10 +340,12 @@ public long getCloudToDeviceMessageCount()
 
     /**
      * Setter for force update boolean
+     * @deprecated  This method currently only validates forceUpdate parameter
      *
      * @param forceUpdate - Boolean controlling if the update should be forced or not
      * @throws IllegalArgumentException if the provided argument is null
      */
+    @Deprecated
```

Comment:
Deprecating this API and others using this since it doesn't serve the purpose as per the documentation. There is nowhere in SDK we do anything special to force update. 

Since this is public, can not remove. Decided to deprecate and add a comment that explains what is expected using this.